packages/core/src/Stores/client-store.js

```diff
@@ -2263,12 +2264,15 @@ export default class ClientStore extends BaseStore {
             this.setIsLoggingIn(true);
 
             const redirect_url = sessionStorage.getItem('redirect_url');
+            const is_next_wallet_enabled = localStorage.getObject('FeatureFlagsStore')?.data?.next_wallet;
```

Comment:
out of curiosity (basically its out of scope for this PR) - right now we determine if Wallets are enabled by the feature flag (which is available synchronously all the time), but what when we gonna go live?

Reply:
We will remove it then